src/Compilers/Core/Portable/Operations/OperationWalker.cs

```diff
@@ -40,454 +37,20 @@ public override void Visit(IOperation operation)
             }
         }
 
-        internal override void VisitNoneOperation(IOperation operation)
+        public override void DefaultVisit(IOperation operation)
         {
             if (operation is IOperationWithChildren operationWithChildren)
             {
                 VisitArray(operationWithChildren.Children);
             }
         }
 
-        public override void VisitBlockStatement(IBlockStatement operation)
```

Comment:
hmm.. as long as I know it didn't break. and API wise, there is no API change. implementation is changed since there is no override, but BAI wise, there is no change.